### ISSUE TITLE
[bug] [updater] "no updater-enabled targets were built" despite finishing nsis bundle

### ISSUE THREAD

**Issue by S1lander** [tauri-apps/plugins-workspace#1684]
I don't know when this broke, but I was able to push updates without any problems on an earlier beta version.
After running `npm run tauri build` I am getting this output:
```
Finished `release` profile [optimized] target(s) in 1m 12s
    Built application at: D:\User\06 Dev\app\src-tauri\target\release\app.exe
    Info Target: x64
    Running makensis.exe to produce D:\User\06 Dev\app\src-tauri\target\release\bundle\nsis\app_1.0.6_x64-setup.exe
    Warn The bundler was configured to create updater artifacts but no updater-enabled targets were built. Please enable one of these targets: app, appimage, msi, nsis
    Finished 1 bundle at:
        D:\User\06 Dev\app\src-tauri\target\release\bundle\nsis\app_1.0.6_x64-setup.exe

    Error Invalid symbol 10, offset 7.
```
This error doesn't make sense, because it wants `app, appimage, msi, nsis` and then it says that it finished the nsis bundle.
I changed the target from `all` to `nsis` after I got this error the first time. Haven't found any solution yet.
Already tried deleting the `target` folder and then `cargo build`.

`Cargo.toml`
```toml
[build-dependencies]
tauri-build = { version = "2.0.0-rc.6", features = [] }

[dependencies]
tauri = { version = "2.0.0-rc.6", features = [] }
tauri-plugin-shell = "2.0.0-rc.2"
tauri-plugin-fs = "2.0.0-rc.0"
tauri-plugin-dialog = "2.0.0-rc.1"
tauri-plugin-updater = "2.0.0-rc.1"
tauri-plugin-process = "2.0.0-rc.0"
serde = { version = "1", features = ["derive"] }
serde_json = "1"
opener = "0.7.0"
showfile = "0.1.1"
tauri-plugin-devtools = "2.0.0-rc"

[dependencies.tauri-plugin-sql]
features = ["mysql"]
version = "2.0.0-rc.0"

[target."cfg(not(any(target_os = \"android\", target_os = \"ios\")))".dependencies]
tauri-plugin-updater = "2.0.0-rc.1"
```

`tauri.conf.json`
```json
{...
    "updater": {
      "windows": {
        "installMode": "passive"
      },
      "pubkey": "SECRETPUBKEY",
      "endpoints": [
        "https://gist.githubusercontent.com/somethingsomething/raw"
      ]
    }
  },
"bundle": {
    "active": true,
    "createUpdaterArtifacts": true,
    "targets": "nsis",
    "icon": [
      "icons/32x32.png",
      "icons/128x128.png",
      "icons/128x128@2x.png",
      "icons/icon.icns",
      "icons/icon.ico"
    ],
    "resources": [
      "SP-Hand.ttf"
    ],
    "licenseFile": "license.txt",
    "windows": {
      "allowDowngrades": true,
      "certificateThumbprint": null,
      "digestAlgorithm": null,
      "nsis": {
        "installerIcon": "icons/icon.ico"
      },
      "timestampUrl": null,
      "tsp": false,
      "webviewFixedRuntimePath": null,
      "webviewInstallMode": {
        "silent": true,
        "type": "embedBootstrapper"
      },
      "wix": null
    }
  }
```

**Comment by S1lander** [comment]
Ok so I just ...
1. switched back to a previous `Cargo.toml`:
1. deleted the `target` folder
1. ran `cargo build`
1. tried `npm run tauri build` again

... and I was still getting the same issue. So I double checked my environment variables and who would've guessed it ... my private key was wrong because of some stupid mistake! Switched back to the new toml and now we're good 😄

TLDR: Double check environment variables when getting any errors like `Error Invalid symbol 10, offset 7.`

I am still getting the weird Error message tho:
`Warn The bundler was configured to create updater artifacts but no updater-enabled targets were built. Please enable one of these targets: app, appimage, msi, nsis`
